templates/base.html

```
@@ -22,126 +22,157 @@
+                        <i class="bi bi-graph-up"></i>
+                        <span>Statistics</span>
+                    </a>
+                    <a class="app-nav__link {% if active_page == 'settings' %}is-active{% endif %}" href="/settings">
+                        <i class="bi bi-gear"></i>
+                        <span>Settings</span>
+                    </a>
                 </div>
+            </nav>
+
+            <div class="app-sidebar__footer">
+                <button class="btn btn-primary w-100 d-none" id="pwa-install-btn" onclick="installPWA()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)